content/docs/command-reference/plots/templates.md

```diff
@@ -0,0 +1,95 @@
+## plot templates
+
+Dump plot templates. Dumps the templates to `.dvc/plots` by default. Note that
+templates can only be used with
+[data-series plots](/doc/command-reference/plots#types-of-plots).
+
+## Synopsis
+
+```usage
+usage: dvc plots templates [-h] [-q | -v] [-o <path>] [{simple,linear,confusion,confusion_normalized,scatter,smooth}]
```

Comment:
looks like it's not scalable, making a list of all available names. Usually you would have a command that can list them. Or name it a `<template>` and in the details list the available plots.

Reply:
We have created this command intentionally this way to not avoid unnecessary subcommands.
https://github.com/iterative/dvc/pull/7108
If we want to modify it we should probably open a new issue in dvc and get back to it.

Reply:
yes, it's more about this command's interface in this case. Feels like this list `{simple,linear,confusion,confusion_normalized,scatter,smooth}` will be growing / changing. Why do we have to specify it on the parser level, it's not very common to limit that large number of targets there.